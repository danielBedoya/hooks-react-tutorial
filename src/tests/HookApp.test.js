import React from 'react';
import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';

describe ('Prueba en hookapp', () => {
    test('Estructura del componente', () => {
        const wrapper = shallow(<HooksApp />);

        expect(wrapper).toMatchSnapshot();
    });
})