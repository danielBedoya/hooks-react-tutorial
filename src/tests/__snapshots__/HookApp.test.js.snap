// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en hookapp Estructura del componente 1`] = `
<div>
  Hola Mundo
</div>
`;
